mods/common/scripts/utils.lua

```diff
@@ -12,3 +12,23 @@ IdleHunt = function(actor)
 		Trigger.OnIdle(actor, actor.Hunt)
 	end
 end
+
+---Adds a new mandatory objective that is localised and announces it via a message.
+---@param player player
+---@param description string
```

Comment:
"key of the translation string"?

Reply:
EmmyLua does not allow to describe it with text.

Reply:
Scratch that, it does. We probably should adapt the API export.